plans/no-rpm.fmf

```diff
@@ -0,0 +1,25 @@
+prepare:
+    - name: dependencies
+      how: install
+      package: [ golang, go-md2man, gpgme-devel, podman-docker ]
+#    - name: enable-linger
+#      how: shell
+#      script: loginctl enable-linger $(whoami)
+
+discover:
+    keep-git-metadata: true
+    how: shell
+    tests:
+        - name: /validate
+          test: >
+            make &&
+            make tools &&
+            make BUILDTAGS="exclude_graphdriver_btrfs" validate-local &&
```

Comment:
If the BTRFS exclusion matters, I suppose the build tag should be applied consistently to anything that builds the Skopeo Go code, incl. `test-integration*`, `test-unit*`, and `bin/skopeo`.